bin/build-localstack-env

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env bash
+
+# DOC: Build a new docker image for running the localstack container
+# (with our customizations for https://github.com/seattle-uat/civiform/issues/2639)
+# and optionally push to Docker Hub if PUSH_LOCALSTACK_IMAGE is set to anything.
```

Comment:
suggestion (blocking): for the `bin/help` to work properly i think each line needs `# DOC: `